app/controllers/admin/routes.py

```diff
@@ -75,9 +73,18 @@ def createEvent(templateid, programid=None):
         eventData.update(request.form.copy())
 
     if program:
-        # TODO need to handle the multiple programs case
         eventData["program"] = program
 
+        if program.contactName and  program.contactEmail:
+            # TODO need to handle the multiple programs case
+            eventData['contactName'] = program.contactName
+            eventData['contactEmail'] = program.contactEmail
+
+        else:
+            # sets contact name and email to CELTS director (admin) if there is none set
+            eventData['contactName'] = 'Ashley Cochrane'
+            eventData['contactEmail'] = 'ashley_cochrane@berea.edu'
```

Comment:
There is already a `celts_admin_contact` address in the config. Let's just use that.

Reply:
Also, having this here will overwrite whatever they put in, won't it? because this is happening after eventData is updated with their submitted form data. We should only be filling these in when we first show them the create event page.

Reply:
resolved